torch/functional.py

```diff
@@ -64,6 +64,26 @@ def chunk(tensor, chunks, dim=0):
     return split(tensor, split_size, dim)
 
 
+def empty_like(input):
+    r"""empty_like(input) -> Tensor
+
+    Returns an uninitialized tensor with the same size as :attr:`input`.
+
+    Args:
+        input (Tensor): the size of :attr:`input` will determine size of the output tensor
+
+    Example::
+
+        >>> input = torch.LongTensor(2,3)
+        >>> input.new(input.size())
+
+        1.3996e+14  1.3996e+14  1.3996e+14
+        4.0000e+00  0.0000e+00  0.0000e+00
+        [torch.LongTensor of size 2x3]
+    """
+    return input.new(input.size())
```

Comment:
nit: IIRC `input.new().reisize_as_(input)` is slightly faster (we don't have to create a Python tuple, and an object for size of each dim).

Reply:
this is going away once we merge Variables and Tensors anyway, I just wrote this as a stop gap because there's no easy way currently to access the VariableFunction if there's no method and no torch function.